bioio_ome_tiled_tiff/reader.py

```diff
@@ -44,66 +63,184 @@ class Reader(BaseReader):
     _fs: "AbstractFileSystem"
     _path: str
 
-    # Required Methods
+    backend: Optional[str] = None
 
-    def __init__(image: Any, **kwargs: Any):
-        """
-        Store / cache certain parameters required for later reading.
+    def _general_data_array_constructor(
```

Comment:
This method isn't static, I think it might fit better near the bottom of the file with the other private methods IMO

Reply:
resolved in [3e46353](https://github.com/bioio-devs/bioio-ome-tiled-tiff/pull/2/commits/3e46353b361a2b17d4edfec8399cafba3914344b)